### ISSUE TITLE
Dbadmin display an empty page instead adminer interface in dolibarr v14.0.3

### ISSUE THREAD

**Issue by edupulpillo** [ksar-ksar/Dolibarr_DbAdmin#1]
Hi.
Dbadmin display an empty page instead adminer interface in dolibarr v14.0.3
Before the dolibarr update it was working well.
The PHP version did not change, is 7.4.25, and Dolibarr is running in Linux.

I am using the administrator user, and this is the result after click on "Database Admin"
![image](https://user-images.githubusercontent.com/7486607/141743645-31963d1b-014a-41ea-a5f5-712763edae4f.png)

**Comment by ksar-ksar** [comment]
Hello,

Can you activate logs and send them to me ?
For me it is working on V14.0.3
![image](https://user-images.githubusercontent.com/35605507/141744246-212c3e13-87d7-403b-9366-85b7811e5d2a.png)

**Comment by ksar-ksar** [comment]
You are using V1.1 right ?

**Comment by edupulpillo** [comment]
Using 1.0 at the moment.
The screenshot that you kindly mention, is the same i had before the update of Dolibarr. 
How can i update the module from version 1.0 to 1.1 ?
Is any procedure available to uninstall completely a module?

**Comment by ksar-ksar** [comment]
Hello,

Just deploy the new release on the admin part
[module_dbadmin-1.1.zip](https://github.com/ksar-ksar/Dolibarr_DbAdmin/files/7539223/module_dbadmin-1.1.zip)

**Comment by edupulpillo** [comment]
I have deactivated the module.
Then using:
 Setup -> Modules/Applications -> Deploy/install external app/module
 I have uploaded the file you kindly you have provide me.
 The module then works fine.
 So i do not know the reason but your previous version 1.0 of the plugin is not working in Dolibarr after the update. BUt version 1.1 of the plugin works very well. I hope this info be useful for you.
 Thank you for your help.
